src/langs/en/messages.ts

```diff
@@ -28,6 +28,18 @@ export default {
     'One item has too many memberships for the operation to proceed',
   [FAILURE_MESSAGES.MEMBER_CANNOT_ACCESS]:
     'A user does not have access to an item',
+  [FAILURE_MESSAGES.MEMBER_ALREADY_SIGNED_UP]:
+    'You already has an account, we will send you an email to login',
+  [FAILURE_MESSAGES.MEMBER_NOT_SIGNED_UP]: 'No account found for this email',
```

Comment:
Aren't some of these messages leaking information for an attacker that would try to brute-force? Sometimes it is better to have a vague "Incorrect login information" message